recipes/activemq-cpp/all/conanfile.py

```diff
@@ -0,0 +1,125 @@
+        export_conandata_patches(self)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("apr/1.7.4")
+        self.requires("openssl/[>=1.0.0]")
+
+    def validate(self):
+        # Always comment the reason including the upstream issue.
+        # INFO: Upstream only support Unix systems. See <URL>
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            raise ConanInvalidConfiguration(f"{self.ref} is not supported on {self.settings.os}.")
+
+    # if a tool other than the compiler or autotools is required to build the project (pkgconf, bison, flex etc)
+    def build_requirements(self):
+        self.tool_requires("libtool/[>=1.5.22]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        apply_conandata_patches(self)
+
+    def generate(self):
+        # inject required env vars into the build scope
+        # it's required in case of native build when there is AutotoolsDeps & at least one dependency which might be shared, because configure tries to run a test executable
+        if not cross_building(self):
+            VirtualRunEnv(self).generate(scope="build")
+        tc = AutotoolsToolchain(self)
+        # autotools usually uses 'yes' and 'no' to enable/disable options
+        def yes_no(v): return "yes" if v else "no"
+        tc.autoreconf_args.extend(["-I", "config", "-I", "m4"])
+        if self.options.shared:
+            tc.configure_args.extend(["--enable-shared"])
+
+        # Handle the fact that the library uses deprecated throws() declarations
+        compiler_std = self.settings.get_safe('self.settings.compiler.cppstd')
+        if compiler_std is None or compiler_std > 14:
+            tc.extra_cxxflags.append('-std=c++14')
+
+        apr_package_folder = self.dependencies.direct_host["apr"].package_folder
+        tc.configure_args.extend([
+                f"--with-apr={apr_package_folder}"
+            ])
+
+        tc.generate()
+        # generate pkg-config files of dependencies (useless if upstream configure.ac doesn't rely on PKG_CHECK_MODULES macro)
+        tc = PkgConfigDeps(self)
+        tc.generate()
+        deps = AutotoolsDeps(self)
+        deps.generate()
+
+    def build(self):
+        autotools = Autotools(self)
+
+        script_folder = "activemq-cpp"
+
+        if conan_version >= Version("2.0.2"):
+            autotools.autoreconf(build_script_folder=script_folder)
+        else:
+            with chdir(self, os.path.join(self.source_folder, script_folder)):
+                mkdir(self, "config")
+                command = "autoreconf --force --install -I config -I m4"
+                self.run(command)
+        
+        autotools.configure(build_script_folder=script_folder)
+        autotools.make()
+
+    def package(self):
+        copy(self, "LICENSE", self.source_folder, os.path.join(self.package_folder, "licenses"))
```

Comment:
https://github.com/apache/activemq-cpp/blob/activemq-cpp-3.9.5/LICENSE.txt
```suggestion
        copy(self, "LICENSE.txt", self.source_folder, os.path.join(self.package_folder, "licenses"))
```

Reply:
Fixed